src/test/scala/helpers/TestHelper.scala

```diff
@@ -35,18 +35,20 @@ object TestHelper {
   }
 
   def getDbSettings() =
-      DbSettings(getDriverFromEnv,      // driver
-          System.getenv("DBCONNSTRING"),  // nrl
-          System.getenv("DBUSER"),        // user
-          System.getenv("DBPASSWORD"),    // password
-          System.getenv("DBNAME"),        // dbname
-          System.getenv("DBHOST"),
-          System.getenv("DBPORT"),
-          System.getenv("GPHOST"),
-          System.getenv("GPPATH"),
-          System.getenv("GPPORT"),
-          getGPLOADEnv(),
-          getIsIncrementalEnv())
+      DbSettings(
```

Comment:
(Not related to this PR)
I think we might want to use a map for `DbSettings`. When adding new settings inside `DbSettings`, we need to to change all the initializations for the case class. A map may be much easier to maintain.